tests/unit/v2/RedisClient.js

```diff
@@ -0,0 +1,21 @@
+const RedisClient = require('../../../libV2/redis');
+
+describe('Test RedisClient', () => {
+    let client;
+
+    beforeEach(() => {
+        client = new RedisClient({});
+        client.connect();
+    });
+
+    afterEach(() => client.disconnect());
+
+    it('should not raise exception if redis backend emits error', () => {
```

Comment:
could add an extra point to the it saying something like "if redis backend emits error without a listener"? Or equivalent as it wasn't quite clear to me at first.

Reply:
done